src/containers/App.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Link } from 'react-router';
-import { Menu } from '../components';
+import { connect } from 'react-redux';
+
+import { StageMenu } from '.';
 
 require('../styles/main.scss');
 
+
+const menuIsOpen = state => state.menu.menuIsOpen;
```

Comment:
This follows the selector pattern, maybe it should live in `selectors` one day, but this looks fine to me.